sprocs/sync_assessments.sql

```diff
@@ -43,7 +43,8 @@ BEGIN
             constant_question_value,
             allow_issue_reporting,
             allow_real_time_grading,
-            require_honor_code)
+            require_honor_code,
+            groupWork)
```

Comment:
Please use the same naming style as surrounding variables (`snake_case` in this case).

Reply:
Just to clarify, this means it will be called `group_work` in some places (e.g., here in the sql) but `groupWork` in other places (e.g., json).